### ISSUE TITLE
Weird rendering of an intersection with Live updates On

### ISSUE THREAD

**Issue by mdykierek** [osmandapp/OsmAnd#16036]
Street [Księdza Marcina Strzody](https://www.openstreetmap.org/way/184730109) in Gliwice is split into 3 separate roads.
Nearby Akademicka split into 2.
Some data doesn't get updated. Crosswalk on Akademicka should now be with traffic signals...

![Screenshot_2022-12-22-15-50-39-460_net osmand plus](https://user-images.githubusercontent.com/27214784/209160290-b497721f-9040-46ba-b84d-76b368eef4c0.jpg)

### Your Environment
OsmAnd Version: OsmAnd+ 4.3.3
Android/iOS version: Android 11
Device model: Xiaomi Mi 9T Pro

**Maps used (online or offline):**   
If you have an issue related to offline maps, tell us the exact name of the map file where the issue occurs and its edition date.

**Comment by scaidermern** [comment]
Are you using the new map rendering engine with OpenGL? If so, try to disable it and check if the issue persists. If this issue is gone then this is likely #14920.

**Comment by mdykierek** [comment]
When I first noticed it, the Strzody street was split into 2. I went back home and made some improvements to the street mapping in OSM (there were some issues with lane count, turn:lanes, etc.) and a couple hours later it was already 3 lines. So I disabled live updates and restarted OsmAnd+. The issue is gone now for both V2 and V3. The crosswalks are OK now even after re-enabling live updates - hmm...
